app/controllers/blogs_controller.rb

```diff
@@ -0,0 +1,59 @@
+class BlogsController < ApplicationController
+
+  before_action :logged_in_user, only: [:new, :edit, :index, :show, :create,]
+
+
+  def index
+
+  end
+
+
+  def show
+
+  end
+
+
+  def edit
+
+  end
+
+  def new
+  	@blog = Blog.new
+  end
+
+  def create	
+  	@blog = Blog.new(params.require(:blog).permit(:title,:caption,:description))
+  	@blog.user_id = current_user.id
+	if @blog.save
+		flash[:success] = "Micropost created!"
+		redirect_to root_url
+	else
+		flash.now[:error] = "Could not save"
+		render '/pages/home'
+	end
+  end
+
+  private
+  			    # Confirms a logged-in user.
+		def logged_in_user
```

Comment:
O.o
I don't understand which code to remove and where to find that line #3 comment.